src/runtime/transaction_execution.zig

```diff
@@ -192,6 +196,7 @@ pub fn loadAndExecuteTransactions(
     environment: *const TransactionExecutionEnvironment,
     config: *const TransactionExecutionConfig,
 ) error{OutOfMemory}![]TransactionResult(ProcessedTransaction) {
+    const program_map = ProgramMap{};
```

Comment:
could do with a comment for where the "real" program_map is supposed to come from